frontend/src/components/ViewActivities/activityfns.js

```diff
@@ -62,8 +63,12 @@ export function sortByDate(tripActivities) {
       activities.set(dateKey, new Set([activity]));
     }
   }
-  // Sort activities by date.
-  let activitiesSorted = Array.from(activities).sort(compareActivities);
+  // Sort activities by date.  
+  let activitiesSorted = Array.from(activities).sort((a, b) => {
+    const aDate = moment.utc(a[0], "YYYY-MM-DD").valueOf();
+    const bDate = moment.utc(b[0], "YYYY-MM-DD").valueOf();
+    return aDate < bDate ? -1 : 1;
+  });
```

Comment:
Isn't this change already made in #122 ? If so remove it in one or the other PR.

Reply:
closed #122